sycl/include/sycl/properties/accessor_properties.hpp

```diff
@@ -109,6 +112,17 @@ template <typename T> struct is_host_accessor : std::false_type {};
 template <typename DataT, int Dimensions, access::mode AccessMode>
 struct is_host_accessor<host_accessor<DataT, Dimensions, AccessMode>>
     : std::true_type {};
+
+template <typename T> struct is_local_accessor : std::false_type {};
+template <typename T, int Dimensions>
+struct is_local_accessor<local_accessor<T, Dimensions>> : std::true_type {};
+
+template <typename T> struct is_image_accessor : std::false_type {};
+template <typename T, int Dimensions, access::mode AccessMode,
+          access::placeholder IsPlaceholder, typename PropertyListT>
+struct is_image_accessor<
+    accessor<T, Dimensions, AccessMode, access::target::image, IsPlaceholder,
+             PropertyListT>> : std::true_type {};
```

Comment:
Can we add and use 

```
template <typename T>
inline constexpr bool is_image_accessor_v = is_image_accessor<T>::value;
```

Reply:
Ok, added it in 135ed67d5a82d2edf5535e7262bbc15f4d3e7f5d